docs/devices/FlashingCertificate/CLI.rst

```diff
@@ -7,7 +7,7 @@ Provisioning using the CLI
 
 .. note::
 
-   Provisioning the certificates and programming the firmware through CLI are not currently supported on WSL 2 since it lacks support for serial devices. 
+   Provisioning the certificates and programming the firmware through CLI are not currently supported on WSL2 since it lacks support for serial devices.
```

Comment:
Product abbreviation is `WSL 2`: https://docs.microsoft.com/en-us/windows/wsl/compare-versions#whats-new-in-wsl-2
```suggestion
   Provisioning the certificates and programming the firmware through CLI are not currently supported on WSL 2 since it lacks support for serial devices.
```